core/lib/types/src/api/en.rs

```diff
@@ -25,7 +25,7 @@ pub struct SyncBlock {
     /// L2 block timestamp.
     pub timestamp: u64,
     /// L1 gas price used as VM parameter for the L1 batch corresponding to this L2 block.
-    pub l1_gas_price: u64,
+    pub l1_gas_price: U256,
     /// L2 gas price used as VM parameter for the L1 batch corresponding to this L2 block.
     pub l2_fair_gas_price: u64,
     /// The pubdata price used as VM parameter for the L1 batch corresponding to this L2 block.
```

Comment:
Same here, why did you change only `l1_gas_price`?

Reply:
Same comment as [this](https://github.com/matter-labs/zksync-era/pull/1324#discussion_r1513527901)